site/content/device/pc-cooling-fan.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "PC Cooling Fan"
```

Comment:
Can you just rename it "fan" ? I mean PC cooling fan is an application of a fan, but any fan can be controlled the same. Also rename the jpg to "fan.jpg"